### PR TITLE
[4.0] Table Column alignment (part2) [a11y]

### PR DESCRIPTION
**Pull request by brianteeman**
com_fields
com_csp
com_installer

Continuation of #21933

### CODE REVIEW

File: administrator/components/com_installer/tmpl/database/default.php [hunk 1]
```diff
@@ -89,29 +89,29 @@
...
-											<td class="text-center">
+											<td>
 												<?php echo $extension->client_translated; ?>
 											</td>
-											<td class="text-center">
+											<td>
 												<?php echo $extension->type_translated; ?>
 											</td>
-											<td class="d-none d-md-table-cell text-center">
+											<td class="d-none d-md-table-cell ">
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/database/default.php]: Remove space

File: administrator/components/com_installer/tmpl/database/default.php [hunk 2]
```diff
@@ -37,37 +37,37 @@
 							<table class="table">
 								<thead>
 									<tr>
-										<td class="nowrap" style="width:1%">
+										<td class="nowrap text-center" style="width:1%">
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/database/default.php]: `nowrap` can be removed to match below.

```
											<td class="text-center">
												<?php echo HTMLHelper::_('grid.id', $i, $extension->extension_id); ?>
											</td>
```

There are other instances like this.

File: administrator/components/com_installer/tmpl/manage/default.php [hunk 3]
```diff
@@ -49,36 +49,36 @@
 								<th scope="col" class="nowrap">
 									<?php echo HTMLHelper::_('searchtools.sort', 'COM_INSTALLER_HEADING_NAME', 'name', $listDirn, $listOrder); ?>
 								</th>
-								<th scope="col" style="width:10%" class="text-center">
+								<th scope="col" style="width:10%">
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/manage/default.php:52]: Remove `nowrap` above.
```
 								<td style="width:1%" class="nowrap text-center">
 									<?php echo HTMLHelper::_('grid.checkall'); ?>
</td>
```

File: administrator/components/com_installer/tmpl/update/default.php [hunk 4]
```diff
@@ -47,16 +47,16 @@
 								<th scope="col" class="nowrap">
 									<?php echo HTMLHelper::_('searchtools.sort', 'COM_INSTALLER_HEADING_NAME', 'u.name', $listDirn, $listOrder); ?>
 								</th>
-								<th scope="col" class="nowrap text-center">
+								<th scope="col" class="nowrap">
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/update/default.php:50]: Remove `nowrap` above.

Comment (brianteeman) [reply administrator/components/com_installer/tmpl/update/default.php]: why?

**Comment by ciar4n** [comment]
`nowrap` is been applied here to all table headers... https://github.com/joomla/joomla-cms/blob/4.0-dev/administrator/templates/atum/scss/vendor/bootstrap/_table.scss#L7

Should either remove this CSS and use the bootstrap `nowrap` utility class everywhere where needed. Or keep it and remove all the `nowrap` classes from table headers.

**Comment by brianteeman** [comment]
Ok cool so I can remove all the nowrap then. Makes it easier for all.

**Comment by brianteeman** [comment]
I will deal with nowrap in its own pr

**Comment by brianteeman** [comment]
@zwiastunsw Can you test this one as well please Stefan so I can move on to the next stage

**Comment by renekreijveld** [comment]
I have tested this item :white_check_mark: successfully on f185c73edc277087bcd0dee92e9abae117133a6a<br /><br />Before patch applied:
Inspected all column headings for presence of class="text-center".
Applied patch
Verified all classes were modified.
Still one problem: administrator/components/com_csp/tmpl/reports/default.php, line 117
Originally contained twice the class "text-center" and only one is removed.
I could NOT test the extensions update view as Joomla didn't discover extension updates.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21938">issues.joomla.org/tracker/joomla-cms/21938</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**